lib/puppet/module_tool/skeleton/templates/generator/Gemfile

```diff
@@ -1,7 +1,14 @@
-source 'https://rubygems.org'
+source ENV['GEM_SOURCE'] || 'https://rubygems.org'
 
 puppetversion = ENV.key?('PUPPET_VERSION') ? "#{ENV['PUPPET_VERSION']}" : ['>= 3.3']
+gem 'metadata-json-lint'
```

Comment:
@hunner do you have a more detailed explanation? Is this a problem in module CI where they use system Ruby and the puppet gem?

@garethr's skeleton uses this for :test (https://github.com/garethr/puppet-module-skeleton/blob/master/skeleton/Gemfile).